### ISSUE TITLE
alert outside of shoes.app blocks

### ISSUE THREAD

**Issue by PragTob** [shoes/shoes4#149]
Hi everyone,

as our awesome @ashbb pointed out, alert can be used outside of the shoes block. The question is: Do we want to preserve that behaviour in shoes4?

I think it is kind of weird to have it outside of the good old Shoes.app block. Also we'd have to monkey patch Object or the main object... which I don't really like. 

Note this doesn't only go for `alert`, but most likely for `ask` and `confirm` as well... and probably all the other popup/dialog methods (color picker and file dialog come to mind if I'm not mistaken,)

Long story short: Do we want to make the backwards incompatible change to only call all those methods from within the Shoes.app block or not?

Cheers + thanks for feedback,
Tobi
